go.mod

```diff
@@ -5,7 +5,14 @@ go 1.21
 replace k8s.io/client-go => k8s.io/client-go v0.28.6
 
 require (
-	github.com/aws/aws-sdk-go v1.49.24
+	github.com/aws/aws-sdk-go v1.50.34
```

Comment:
Is the previous version still used somewhere?

Reply:
it will be addressed in another PR